src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -140,6 +143,23 @@ def reset_arg(arg, value):
                 reset_arg(arg, '0')
             reset_arg('compression_algorithm', 'unset')
 
+    def _wait_for_pgs(self, pool):
+        current_pool = self._get(pool)
```

Comment:
I'm missing any documentation here. Is ``pool`` a string or int?

Reply:
`pool` is a string and I will add the missing documentation - Thanks for the hint :)